docs/help-center/using-famedly/chats.mdx

```diff
@@ -284,10 +285,11 @@ You can see the status of other users on the profile pictures in the following p
 
 ### Which symbol shows which status?
 
-**Active** &rarr; Green dot on the profile picture
-**Away** &rarr; Grey dot on the profile picture  
+**Active** &rarr; Green dot on the profile picture  
+**Away** &rarr; Grey dot on the profile picture &rarr; Also shown if the user has disabled sending presences 
 
-Also you can see the status as text in the title bar of 1:1 chats. In previous versions "online" has been displayed instead of "active".
+Also you can see the status as text in the title bar of 1:1 chats. In previous versions "online" has been displayed instead of "active". 
+If the user has set up an optional status message, this will be shown instead of "active". 
```

Comment:
If this is true, I think we need to change the wording of a status message. It is now an OOO message. A status message may also be 'I just got married' or 'I got promoted to STATUS'. This would not mean I am inactive. Maybe @Krzysztof2512 can improve this a bit, if it is an OOO reply, we should call it that.